### PR TITLE
Add the ability to embed orgs, fix #495

### LINKED ISSUE #495: Add the ability to load organizations with embeds
## Detailed Description

We can already load datasets and territories, it's time for orgs.

### PR DESCRIPTION
**Pull request by davidbgk**
Please review  https://github.com/etalab/udata-gouvfr/pull/115 too

### CODE REVIEW

File: js/widgets.js [hunk 1]
```diff
@@ -43,26 +50,28 @@ function extractURLs (selector) {
...
+    {'base': 'O', 'letters': /[Ô]/g},
+    {'base': 'OE', 'letters': /[Œ]/g},
+    {'base': 'U', 'letters': /[ÛÙ]/g},
+    {'base': 'a', 'letters': /[à]/g},
+    {'base': 'e', 'letters': /[éêè]/g},
+    {'base': 'i', 'letters': /[ï]/g},
+    {'base': 'o', 'letters': /[ô]/g},
+    {'base': 'oe', 'letters': /[œ]/g},
+    {'base': 'u', 'letters': /[ûù]/g},
+    {'base': '', 'letters': /[\])}'’,.\-—–[{(]/g}
```
Comment (noirbizarre) [inline js/widgets.js]: I don't know if both writings are equivalent but this one is definitively easier to understand ;) :+1:

Comment (davidbgk) [inline js/widgets.js]: Not exactly equivalent given that I removed non-French diacritics but definitely easier to enrich and understand ;-)

File: js/widgets.js [hunk 2]
```diff
@@ -255,59 +261,61 @@ function embedDatasets (territories, datasets) {
...
+function filterDatasets (element, event, datasets, initialDatasets) {
   const searchValue = event.target.value
   if (searchValue.length < 3) {
-    territoryElement.innerHTML = initialDatasets
+    element.innerHTML = initialDatasets
     return
   }
   const scoredDatasets = datasets.map((dataset) => {
-    const title = removeDiacritics(dataset.querySelector('.udata-title').innerText)
+    const title = removeDiacritics(dataset.querySelector('.udata-title a').title)
```
Comment (noirbizarre) [inline js/widgets.js]: Reusing dedicated attribute :+1:

File: udata/templates/territories/territory.html [hunk 3]
```diff
@@ -46,5 +46,5 @@
 {% block extra_js %}
     <script src="{{ static('vue-common.js') }}"></script>
     <script src="{{ static('territory.js') }}"></script>
-    <script src="{{ static('widgets.js', _burst=False) }}" id="udata" async defer onload="udataScript.load()"></script>
+    <script src="{{ static('widgets.js', _burst=False) }}" id="udata" async defer onload="udataScript.loadDatasets()"></script>
```
Comment (noirbizarre) [inline udata/templates/territories/territory.html]: Does the embed code (the one proposed to the user with the button) need the same modification ?

**Comment by noirbizarre** [comment]
r+wc

File: udata/templates/territories/territory.html (hunk 3 above)
Comment (davidbgk) [inline udata/templates/territories/territory.html]: Yes, and that's what is done here: https://github.com/opendatateam/udata/pull/496/files/4e22e4e8bddb70cd7b81a37aba7581a1430f93b6#diff-963e0f551bd557273268a73a942c30e7R168

File: js/widgets.js (hunk 1 above)
Comment (noirbizarre) [inline js/widgets.js]: Fine to me but they're should be a comment explaining this

File: udata/templates/territories/territory.html (hunk 3 above)
Comment (noirbizarre) [inline udata/templates/territories/territory.html]: I was refering to:
- https://github.com/opendatateam/udata/blob/master/js/templates/widgets/integrate-popover.hbs
- https://github.com/etalab/udata-gouvfr/blob/master/udata_gouvfr/templates/faq/developer.html#L57-L61

Comment (davidbgk) [inline udata/templates/territories/territory.html]: Oh I totally forgot the first, thanks! The latter has already been taking care of though https://github.com/etalab/udata-gouvfr/pull/115

**Pull request merged**